examples/ansible_nav.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: cloud.redhat.com/v1alpha1
+kind: Bundle 
+metadata:
+    name: ansible-navigation
+spec:
+  id: ansible-navigation
+  title: ansible
+  envName: env-boot
+  appList:
+  - ansibleDashboard
+  - automationHub
+  - automationAnalytics
+  - documentation
+  - registration
+  - remediations
+  - tasks
+  customNav:
+  - groupId: operations # Bundle
+    title: "Red Hat Insights"
+    navItems:
+    - title: "Advisor" # Leaf
+      expandable: true
+      routes: # Can't have routes on leaf
```

Comment:
I'm confused by the "can't have routes on leaf" comment here -- isn't this example in fact showing routes on a leaf? :)

Reply:
Lol I annotated that to determine what was wrong. Lemme clean it up.